keps/sig-cloud-provider/1959-service-lb-class-field/README.md

```diff
@@ -186,6 +186,10 @@ E2E tests:
 * Feature gate is on by default.
 * E2E tests checking that default load balancer implementation ignores LoadBalancer type of Services when `loadBalancerClass` set.
 
+#### GA
+
+* Ensure integration/E2E tests continue to work and provide sufficient coverage.
```

Comment:
Not sure there's much more to do than this going to GA, the API change is pretty minimal, and use of the API only impacts external load balancer implementations.